module/zfs/mmp.c

```diff
@@ -392,6 +391,30 @@ mmp_thread(void *arg)
 		else
 			next_time = start + MSEC2NSEC(MMP_DEFAULT_INTERVAL);
 
+		/*
+		 * Don't allow fail_intervals larger than import_intervals,
```

Comment:
Would this be better implemented in a callback, so you can reject bad values at the time the user tries to set them?  One such example is: 
* 0582e40 Add callback for zfs_multihost_interval